libraries/chain/controller.cpp

```diff
@@ -252,6 +252,11 @@ struct controller_impl {
                            { check_protocol_features( timestamp, cur_features, new_features ); }
       );
 
+      // Might be too late?
+      if ( cfg.backing_store == backing_store_type::ROCKSDB ) {
```

Comment:
Is there a reason we don't just do this in the constructor for kv_db?  It is passed cfg.

Reply:
I will change to use constructor of kv_db.